### DISCUSSION TITLE
Mineral scanner upgrade

### DISCUSSION THREAD

**Discussion by someguythatplaysbarotrauma** [FakeFishGames/Barotrauma#8466] [Ideas and suggestions]
So, have you ever gone to an outpost, looking for supplies like steel, or copper or anything in general, and then you go to the outpost shop and found that the shop either doesn't have the material(s) you need, or they just don't have enough of what you need, and you decide you want to find materials so you have a surplus or you get the materials for free, then you have to scan for minerals, but the problem is, the only way to scan for materials is a handheld sonar. You can't just use the sonar inside of your sub, since it doesn't work that way, and the battery drains very fast. The solution to this would be an upgrade. A mineral scanner upgrade. It would work like a handheld sonar, where you can switch the mineral scanning off or on, but the catch is that it would take extra electricity to run the scanner. This way, the upgrade is halfway balanced. The way to almost completely balance it would be to make the upgrade take a lot of money, or you would have to buy an expensive item, and then you would have to install it onto your sonar, so then there's like a D.I.Y. mechanic almost, so that is my idea, you don't have to like it, but hey it's an idea.

**Comment by Sevenecho** [comment]
Never had problems of such sort. There're lots of accumulators you get here and there, so you even start selling them. Just keep a stack for an active sonar searching and set for recharge when needed.

**Reply by someguythatplaysbarotrauma** [reply]
well it would be a nice thing to have, and since it could get you a ton of extra resources, it would definitely cost a lot of marks

**Reply by someguythatplaysbarotrauma** [reply]
yeah also I edited the discussion to be a different scenario

**Comment by leonard84** [comment]
Interestingly enough the big sonar monitor can have a mineral scanner mode, but that doesn't work as long as steering is connected. So I guess this was deliberately removed or has always been the case. As for handheld sonar, get fulgurium batteries and upgrade them to better quality by recrafting them. Also don't keep the handheld sonar in active mode constantly but send out a pulse and regular intervals. 

And for MP matches have your gunners on the lookout, as they can also spot plants which don't show up on the _mineral_ scanner.

**Reply by someguythatplaysbarotrauma** [reply]
1: I only play singleplayer 2: the handheld sonar doesn't work inside the sub

**Reply by Sevenecho** [reply]
2: the handheld sonar doesn't work inside the sub
That is not true. I'm the captain, I pick the scanner, activate it, turn the minerals switch, and steer near the sea bottom, watching both locators. When I see something, that's a full stop for me and a mining request for my crew.

**Reply by someguythatplaysbarotrauma** [reply]
well it doesn't work inside the sub when I use it

**Reply by James-E-A** [reply]
@someguythatplaysbarotrauma You have to zoom out. (And you have to be close enough to the walls/caves/etc that it's in-range.)

The handhelds have pretty puny ranges, and hulls block sonar, but the _mineral scanner itself_ pierces hulls perfectly.

**Comment by Next-Door-Tech** [comment]
This probably ought to be closed, since sub sonar mineral scanners are now a thing.
